src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
```

Comment:
The message should be more informative and indicate the specific type of Machine that is working. Consider including the class name 'Bulldozer' in the message to make it clear.